price-feeder/oracle/provider/mock.go

```diff
@@ -118,3 +118,32 @@ func (p MockProvider) GetCandlePrices(pairs ...types.CurrencyPair) (map[string][
 func (p MockProvider) SubscribeCurrencyPairs(pairs ...types.CurrencyPair) error {
 	return nil
 }
+
+// GetAvailablePairs return all available pairs symbol to susbscribe.
+func (p MockProvider) GetAvailablePairs() (map[string]struct{}, error) {
+	resp, err := http.Get(p.baseURL)
+	if err != nil {
+		return nil, err
+	}
+	defer resp.Body.Close()
+
+	csvReader := csv.NewReader(resp.Body)
+	records, err := csvReader.ReadAll()
+	if err != nil {
+		return nil, err
+	}
+
+	availablePairs := make(map[string]struct{}, len(records[1:]))
```

Comment:
Can add a comment that the [1:] is removing the first/title row of the csv from the google spreadsheet

Reply:
done at https://github.com/umee-network/umee/pull/649/commits/07bdb9d3352b36baa1dddf734443b7ae16af93ab